src/common/getAgesOfFamily.ts

```diff
@@ -0,0 +1,49 @@
+export const getCurrentYear = (): number => {
+    const currentDate = new Date();
+    return currentDate.getFullYear();
+};
+export const getCurrentMonth = (): number => {
+    const currentDate = new Date();
+    return currentDate.getMonth() + 1;
+};
+
+export const isChildUsingBirthYear = (birthYear: number): boolean => {
+    return getCurrentYear() - birthYear <= 16;
+};
+
+export const getChildAgeUsingBirthYearAndMonth = (
+    birthYear: number,
+    shortHand: boolean,
+    birthMonth?: number
```

Comment:
it might make sense to have parameters in this order and make birthMonth nullable? Or at this point, we could just pass a `Person` as a parameter instead?
```suggestion
    birthYear: number,
    birthMonth: number | null
    shortHand: boolean,
```